src/main/java/com/dislinkt/post/model/Person.java

```diff
@@ -0,0 +1,168 @@
+package com.dislinkt.post.model;
+
+import java.util.Date;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.UUID;
+
+import javax.persistence.*;
+
+import com.dislinkt.post.enums.Gender;
+import com.dislinkt.post.enums.ProfilePrivacy;
+
+@Entity
+public class Person {
+
+    @Id
+    @GeneratedValue(generator = "UUID")
```

Comment:
I'm not sure if it should be generated value since this field should be ONE-ONE with an ID of the user entity (auth microservice). So you will have it provided and resolved from gateway.

Reply:
But in other classes, you are missing @GeneratedValue. Does it work anyways by default?